pediatric_rootlets/crop_composed_images.py

```diff
@@ -0,0 +1,95 @@
+import os
```

Comment:
Please add a brief description what the script is doing
```suggestion
"""
This script ...
"""

import os
```

Reply:
Resolved in [a9ac697](https://github.com/ivadomed/model-spinal-rootlets/pull/53/commits/a9ac6970c37b7102e6118a3e7858ba06cd9ecea4).